framework/src/main/java/org/checkerframework/common/value/ValueAnnotatedTypeFactory.java

```diff
@@ -164,11 +167,23 @@
     /** Helper class that holds references to special methods. */
     private final ValueMethodIdentifier methods;
 
+    /** The property file handler. */
+    public PropertyFileHandler propertyFileHandler = null;
+
+    /**
+     * Create a new ValueAnnotatedTypeFactory.
+     *
+     * @param checker the checker to use
+     */
     public ValueAnnotatedTypeFactory(BaseTypeChecker checker) {
         super(checker);
 
         reportEvalWarnings = checker.hasOption(ValueChecker.REPORT_EVAL_WARNS);
         Range.ignoreOverflow = checker.hasOption(ValueChecker.IGNORE_RANGE_OVERFLOW);
+        if (checker.hasOption(ValueChecker.HANDLE_PROPERTY_FILE)) {
```

Comment:
I think setting a checker option is a very inconvenient way for users to configure a subchecker.
Can you again think about how we can improve the subchecker mechanism to allow setting options more flexibly?

Reply:
Sure, no problem. I will keep thinking about it. Should we do it within this pull request?

Reply:
Punting on this for now.